test/test_subsonic.py

```diff
@@ -54,19 +36,24 @@ def test_start_scan(self):
 
     @responses.activate
     def test_url_with_extra_forward_slash_url(self):
-        config["subsonic"]["url"] = "http://localhost:4040/contextPath"
+        self.config = {'url': 'http://localhost:4040/contextPath'}
```

Comment:
I think the old way above is the right way to set config options for a test—that is, overwrite individual values. Doing `self.config = ...` will replace the entire configuration object (which is not a dict ordinarily, but a Confuse view), whereas what you want is to change the _contents_ of the configuration.